### PR TITLE
Nine: a few doc fixes and code fixes

### PR DESCRIPTION
**Pull request by jaredgrubb**
I have finally got some time to play with nine and had some suggestions on doc-fixes and fixed one bug on the module name for the ldap stuff.

### CODE REVIEW

**Comment by rutsky** [comment]
LGTM, docs changes like these makes Buildbot much more end-user friendly, thanks!

However tests are not passing (`buildbot.test.unit.test_steps_master.TestSetProperty.test_simple`).

File: master/docs/conf.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
+    'sphinx.ext.autodoc',
```
Comment (rutsky) [inline master/docs/conf.py]: Can you explain why this needed and this is useful?

Comment (jaredgrubb) [inline master/docs/conf.py]: When I ran "make docs", it failed inside Pillow and said that this module was not recognized. Adding this line fixed that issue.

**Comment by jaredgrubb** [comment]
Fixed the unit test. inlineCallbacks only works on a function that has a yield in it. I should have run them. Thank goodness someone invented CI :)

**Comment by rutsky** [comment]
LGTM.

@jaredgrubb can you take a look at http://trac.buildbot.net/ticket/3371? Is your commit fixes it?

**Comment by jaredgrubb** [comment]
Yes, this fixes the issues in that trac:
 - the typo in the module name (trailing 's')
 - and "RemoteUserAuth(userInfoProvider=userInfoProvider)," to fix the "lower()" issure reporter (since the constructor needs the param by-name to work right).

**Pull request merged**